docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
++
+[source,shell]
+----
+$ terraform init
+----
+When the Terraform provider has been initialized, you see a confirmation message.
+
+[id="proc-using-terraform_{context}"]
+=== Using the RHOAS Terraform provider
+
+Resources are the most important element in the Terraform language. Each resource block in a Terraform provider describes one or more infrastructure objects. For {product-long-kafka}, such infrastructure objects might include Kafka instances, service accounts, Access Control Lists (ACLs), and topics. The procedure that followS show what resources you can add to your Terraform configuration file to create a Kafka instance and its associated resources such as service accounts and topics.
+
+.Prerequisites
+
+* You have a Red Hat account.
+* You have an offline token that authenticates the Terraform resources with the OpenShift Application Services API.
+
+[NOTE]
+====
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and you can find it at https://cloud.redhat.com/openshift/token. As the offline token is a sensitive value that varies between environments it is best specified as an `_OFFLINE_TOKEN_` environment variable when running `terraform apply` in a terminal. To set this environment variable, enter the following command in a terminal window, replacing <offline_token> with the value of the offline token:
+[source, subs="+quotes"]
+----
+export OFFLINE_TOKEN=<offline_token>
+----
+====
+
+.Procedure
+
+. Open the `main.tf` file in your IDE for editing.
+. Enter the example rhoas_kafka resource. This example uses the `"my-instance"`  identifier and creates a Kafka instance called `my-instance`. The only required fields are the `name`, `billing_model` and `plan` fields. The following values are set by default when you run `terraform apply`.
++
+.. `cloud provider`
+.. `region`
++
+All other information for the instance is provided by the Kafka APIs. The generated bootstrap server displays in the terminal as an output.
++
+.Example `rhoas_kafka` resource
+[source]
+----
+resource "rhoas_kafka" "my-instance" {
+  name = "my-instance"
+  plan = "developer.x1"
+  billing_model = "standard"
+}
+  output "bootstrap_server_my-instance" {
+    value = rhoas_kafka.my-instance.bootstrap_server_host
+}
+
+----
+. After creating your Kafka instance, you create a service account. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. In the `main.tf` file, enter the example `rhoas_service_account` resource to create a service account. This example uses the `"my-service-account"` identifier and creates a service account called my-service-account. The generated client ID displays in the terminal as an output.
```

Comment:
We're kind of saying the same thing twice here. I'd remove the first sentence. I'd also remove the word "first" from the second sentence.

Reply:
Same question here about the verb "enter" - do we mean _scroll down_ to that resource in the file?